docs/NOTIFICATIONS.md

```diff
@@ -22,7 +22,34 @@ SPRING_MAIL_PASSWORD="pass"
 
 ### Email template
 Templates ensure consistent and automated communications for various functions. 
-We are using [Thymeleaf](https://www.thymeleaf.org/) as the template engine, the email templates are stored in the `resources/templates/` directory.
+We are using [Thymeleaf](https://www.thymeleaf.org/) as the template engine, which allows us to create dynamic and customizable email content.
+
+#### Template location
+The email template is stored in the `src/main/resources/templates/` directory. It is written in HTML with Thymeleaf syntax, allowing you to include dynamic content using expressions, conditionals, and loops.
+
+#### Editing the template
+ - To edit the template, open the `email-notification.html` file.
```

Comment:
maybe also provide instructions on how to upload a custom file for docker containers?